net/yggdrasil/files/ygguci

```diff
@@ -81,7 +81,7 @@ function UCI.get()
       table.insert(obj.InterfacePeers[s["interface"]], s.uri)
   end)
 
-  local node_info = cursor:get_all("yggdrasil", "NodeInfo")
```

Comment:
OK -- If we want to try to match yggdrasil.config to uci let's keep this patch out of the mix.

Reply:
Just leave NodeInfo the way it is, and we'll try to keep the same case between config and UCI